<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(239, 245, 255);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>561</height>
    </rect>
   </property>
   <property name="title">
    <string>Mailing</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_from">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255); 
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255); 
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_to">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255); 
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_subject">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>541</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255); 
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_mail">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>541</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255); 
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>510</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 195, 15);
border: 2px solid gray; 
border-radius: 10px; 
padding: 0 8px; 
selection-background-color: darkgray;</string>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Type your mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Objet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Destinataire</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exp√©diteur</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot de passe</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(232, 232, 232);
border-radius: 10px; 
padding: 0 8px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
